### PR TITLE
test for invalid filename and nonhappy path prop files

### PR DESCRIPTION
**Pull request by cfarmer-fearless**
Added a new test in `MetadataVerify.kt` for invalid filename characters.
Also added usecase specific properties files for non-happy path metadata verify tests.

### CODE REVIEW

File: tests/integration/kotlin/src/test/kotlin/MetadataVerify.kt [hunk 1]
```diff
@@ -40,24 +53,12 @@ class MetadataVerify {
...
     @Test(groups = [Constants.Groups.METADATA_VERIFY], expectedExceptions = [ProtocolException::class])
     fun shouldReturnErrorWhenEventNotProvided() {
-        val metadata = hashMapOf(
-            "filename" to testFile.name,
-            "meta_destination_id" to TEST_DESTINATION,
-            "meta_ext_source" to "INTEGRATION-TEST"
-        ) as HashMap<String, String>
-
-        uploadClient.uploadFile(testFile, metadata)
+        uploadClient.uploadFile(testFile, metadataNoEvent)
```
Comment (USZ7) [inline tests/integration/kotlin/src/test/kotlin/MetadataVerify.kt:61]: Don't we need assertions for non-happy path tests?

File: tests/integration/kotlin/src/test/kotlin/MetadataVerify.kt [hunk 2]
```diff
@@ -70,4 +71,9 @@ class MetadataVerify {
 
         uploadClient.uploadFile(testFile, metadata)
     }
+
+    @Test(groups = [Constants.Groups.METADATA_VERIFY], expectedExceptions = [ProtocolException::class])
+    fun shouldReturnErrorWhenFilenameContainsInvalidChars() {
+        uploadClient.uploadFile(testFile, metadataInvalidFilename)
```
Comment (USZ7) [inline tests/integration/kotlin/src/test/kotlin/MetadataVerify.kt:77]: same here.

File: tests/integration/kotlin/src/test/kotlin/MetadataVerify.kt (hunk 1 above)
Comment (cfarmer-fearless) [reply tests/integration/kotlin/src/test/kotlin/MetadataVerify.kt:61]: The assertion comes in the `exceptedExceptions` array in the `@Test` annotation of the test method.

File: tests/integration/kotlin/src/test/kotlin/MetadataVerify.kt (hunk 2 above)
Comment (cfarmer-fearless) [reply tests/integration/kotlin/src/test/kotlin/MetadataVerify.kt:77]: The assertion comes in the `exceptedExceptions` array in the `@Test` annotation of the test method.

File: tests/integration/kotlin/src/test/kotlin/MetadataVerify.kt (hunk 1 above)
Comment (USZ7) [reply tests/integration/kotlin/src/test/kotlin/MetadataVerify.kt:68]: > The assertion comes in the `exceptedExceptions` array in the `@Test` annotation of the test method.

Got it!

**Pull request merged**